carbon-design-system/carbon-for-ibm-dotcom
# feat(card-cta): allow video poster image to be turned off

### Related Ticket(s)

DDS Consulting Issue: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/7063
HC JIRA Ticket: https://jsw.ibm.com/browse/HC-2213 

### Description
- This PR adds a new boolean prop called `no-poster` that allows adopters to turn off the poster image that gets pulled down from the Kaltura video service.

### Design & Functional Specs
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/CTA#video-data-logic-2


### Changelog

**New Components/Props**

- `no-poster` - this prop/attribute allows adopters to turn off the poster image that gets pulled down from the Kaltura video service.

**Removed**

- N/A

### Testing Instructions

![Screen Shot 2021-09-23 at 10 02 52 AM](https://user-images.githubusercontent.com/1815714/134522214-357e1190-8d19-4b4d-968c-39ec4daa4d5d.png)

- Visit the `CTA` UI component with Card variant - `?path=/story/components-cta--card`
- Adjust the CTA type to `Video` and check the `No Video Poster` boolean field.
